crates/viewer/re_viewer_context/src/viewer_context.rs

```diff
@@ -163,6 +163,43 @@ impl ViewerContext<'_> {
         self.selection_state.selected_items()
     }
 
+    /// Returns if this item should be displayed as selected or not.
+    ///
+    /// This does not always line up with [`Self::selection`], if we
+    /// are currently loading something that will be prioritized here.
+    pub fn is_selected(&self, item: &Item) -> bool {
```

Comment:
The semantics are a bit scary. Is calling this `is_selected_or_loading` too much of a stretch? 🤔 

Diverging the selection state from what is shown as selection sounds like asking for lotsa future trouble - why isn't the thing then somehow pinned on the selection state :/
I believe we should try to either sharpen semantics or find a better solution

Reply:
Renaming this sounds good unless I find some better way to express loading items